rudder-web/src/main/scala/com/normation/rudder/web/model/DirectiveFieldEditors.scala

```diff
@@ -176,10 +175,10 @@ class SelectField(val id: String, items: Seq[ValueLabel]) extends DirectiveField
     val valuesByLabel = items.map(_.reverse.tuple).toMap
     val labelsByValue = items.map(_.tuple).toMap
 
-    val defaultLabels = values.map(labelsByValue(_))
+    val defaultLabels = values.map(labelsByValue.getOrElse(_,""))
```

Comment:
Not sure a warning will serve anything but spoil the logs. Perhaps in debug, if somebody is trying to understand a related problem. But agree, the model should not even authorise incorrect values to reach that point.